examples_src/vector-wfs.js

```diff
@@ -5,31 +5,29 @@ goog.require('ol.layer.Tile');
 goog.require('ol.layer.Vector');
 goog.require('ol.loadingstrategy');
 goog.require('ol.source.BingMaps');
-goog.require('ol.source.ServerVector');
+goog.require('ol.source.Vector');
 goog.require('ol.style.Stroke');
 goog.require('ol.style.Style');
 goog.require('ol.tilegrid.XYZ');
 
-var vectorSource = new ol.source.ServerVector({
-  format: new ol.format.GeoJSON(),
+var vectorSource = new ol.source.Vector({
   loader: function(extent, resolution, projection) {
     var url = 'http://demo.boundlessgeo.com/geoserver/wfs?service=WFS&' +
         'version=1.1.0&request=GetFeature&typename=osm:water_areas&' +
         'outputFormat=text/javascript&format_options=callback:loadFeatures' +
         '&srsname=EPSG:3857&bbox=' + extent.join(',') + ',EPSG:3857';
-    $.ajax({
-      url: url,
-      dataType: 'jsonp'
-    });
+    $.ajax({url: url, dataType: 'jsonp'});
   },
-  strategy: ol.loadingstrategy.createTile(new ol.tilegrid.XYZ({
+  strategy: ol.loadingstrategy.tile(new ol.tilegrid.XYZ({
     maxZoom: 19
-  })),
-  projection: 'EPSG:3857'
+  }))
 });
 
 var loadFeatures = function(response) {
-  vectorSource.addFeatures(vectorSource.readFeatures(response));
+  var geojsonFormat = new ol.format.GeoJSON();
+  var features = geojsonFormat.readFeatures(response,
+      {featureProjection: 'EPSG:3857'});
```

Comment:
I meant that I thought it could be changed to this:
```js
var loadFeatures = function(response) {
  var geojsonFormat = new ol.format.GeoJSON();
  var features = geojsonFormat.readFeatures(response);
  vectorSource.addFeatures(features);
};
```